java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -441,14 +447,19 @@ private API getXPAPI(
    * 3. Unqualified enum constant with method call (e.g. "STALE_FLAG.getKey()")
    * 4. Qualified enum name with method call (e.g. "TestExperimentName.STALE_FLAG.getKey()")
    */
-
-  private Matcher<ExpressionTree> isMatchingEnumFieldValue(VisitorState state) {
+  private boolean isMatchingEnumFieldValue(ExpressionTree tree, VisitorState state) {
+    // Handles - Unqualified import enum constant
     Matcher<ExpressionTree> enumFieldValueMatcher =
         symbolMatcher(
             (sym, st) ->
                 isEnumConstantMatchingFlagName(
                     sym.getSimpleName().toString(), ASTHelpers.enclosingClass(sym), state));
-    return anyOf(enumFieldValueMatcher, contains(ExpressionTree.class, enumFieldValueMatcher));
+    // Handles - the other three scenarios
```

Comment:
I think the first matcher handles both unqualified and qualified enum constant. No? At any rate, the method invocation matcher will only handle 3 & 4.

Reply:
Yes, this is a mistake in my comment. Thanks for pointing it out.